packages/viewer/src/lib/scenes/scene.ts

```diff
@@ -259,6 +270,28 @@ export class Scene {
     );
   }
 
+  /**
+   * Applies the specified features of the provided scene view state to the scene.
+   */
+  public async applyPartialSceneViewState(
```

Comment:
Should this be a separate method, or a modification to the existing method? @amvertex @jdm717 Any thoughts?

Reply:
Beyond this, lgtm!

Reply:
I personally liked it as a separate method, but it would be simple to change it to a modification of the existing method.

Reply:
I think I'd lean toward the separate method for now as well. Makes it a little more discoverable, plus I think this parameter order makes more sense than having the `SceneExecutionOptions` second, so it'd be a breaking change to the `applySceneViewState` method